docs/install.md

```diff
@@ -2,6 +2,23 @@
 
 Use this page to add the component to an existing Kubernetes cluster.
 
+## Pre-requisites
+
+1. A Kubernetes cluster (_if you don't have an existing cluster_):
+   ```bash
+   # Example cluster creation command on GKE
+   gcloud container clusters create $CLUSTER_NAME \
+     --zone=$CLUSTER_ZONE
+   ```
```

Comment:
oh nice call adding this @cagiti !! :D

We might want to include a bit more detail here about some of the cluster requirements (or link somewhere that has the details), since you need to have a certain version of k8s (right now Kubernetes version 1.11 or later) and also you'll need a cluster of a minimum size (4 vCPU nodes (n1-standard-4))

More details in other docs at:
* https://github.com/knative/build-pipeline/blob/master/DEVELOPMENT.md#kubernetes-cluster
* https://github.com/knative/docs/blob/master/install/Knative-with-GKE.md#creating-a-kubernetes-cluster